code/index.html

```diff
@@ -3,28 +3,149 @@
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Project 3</title>
-    <!-- dont forget to add a css file and link it here! -->
-  </head>
-  <body>
-    <!-- video or image as a header is cool :) -->
+    <link rel="stylesheet" href="./style.css">
 
-    <!-- Signup form -->
+    <script src="https://kit.fontawesome.com/4bfe1de3df.js" crossorigin="anonymous"></script>
 
-    <section class="accordion">
-      <!-- Accordion -->
-      <h1>FAQ</h1>
 
-      <div class="question" id="section1">Section 1</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Quicksand:wght@300;400;500;600&display=swap" rel="stylesheet">
+    <title>Bakery online</title>
+  </head>
+  <body>
+
+    <!--HEADER-->
+    <header>
+      <picture>
+        <source srcset="./assets/mobile.jpeg" media="(max-width: 768px)" />
+        <source srcset="./assets/tablet.jpeg" media="(max-width: 900px)" />
+        <img class="header__img" src="./assets/desktop.jpeg" alt="cupcakes" />
+      </picture>
+      <div class="header__title">
+        <h1 class="header__content">
+          Sweet & Pretty baking online
+        </h1>
+        <a class="header__button" href="#sign-up">
+          Sign up
+        </a>
       </div>
+    </header>
 
-      <div class="question" id="section2">Section 2</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
+    <!-- SIGN UP FORM -->
+    <section class="form">
+      <div class="form__box">
+        <div class="form__box-left"></div>
+        <div class="form__box-right">
+          <a class="form__title" name="sign-up">
```

Comment:
No need for `name` attribute in anchor - this will not add any functionality. `name` attributes are reserved for form inputs